packages/react-sdk/src/css/css.ts

```diff
@@ -73,21 +81,25 @@ export const generateCssText = (data: Data, options: CssOptions) => {
     }
     const rules = getPresetStyleRules(component, presetStyle);
     for (const [selector, style] of rules) {
-      engine.addStyleRule(selector, { style });
+      globalSheet.addStyleRule(selector, { style });
     }
   }
 
   const styleRules = getStyleRules(styles, styleSourceSelections);
   for (const { breakpointId, instanceId, state, style } of styleRules) {
-    engine.addStyleRule(
-      `[${idAttribute}="${instanceId}"]${state ?? ""}`,
-      {
-        breakpoint: breakpointId,
-        style,
-      },
-      createImageValueTransformer(assets, options)
-    );
+    const transformer = createImageValueTransformer(assets, options);
+    const styleRule = {
+      breakpoint: breakpointId,
+      style,
+    };
+    // @todo add support for both regular and atomic style sheets
+    //sheet.addStyleRule(
+    //  `[${idAttribute}="${instanceId}"]${state ?? ""}`,
+    //  styleRule,
+    //  transformer
+    //);
+    const { classes } = atomicSheet.addStyleRule(styleRule, transformer);
```

Comment:
regular could have the same api and bind to classes instead of attributes.

Reply:
its now mostly the same api (rule, string, transformer) though in regular string is any selector, where in atomic its a selector suffix, since class selector is being generated

Reply:
Just realized what you meant, the reason it was over the attribute so that there is no need to add a class name, since there was already an id there anyways and classname would correspond the  attribute, but I guess we could change that ....

Reply:
with the id its kindof nice to debug as you can know which css rule applies to which instance directly from css rule declaration, maybe we could keep the id and use it as a class: .{id} {}

Reply:
hmm, the same api will only work if we generate the class, not use the id, otherwise we need a different arguments interface to pass the ids

Reply:
we could use the same hashing as with atomic ofc

Reply:
hmm maybe its a good idea to extend the interface to always pass along instance id

Reply:
thought a bit more about it, I think its not worth it ... but we can do it any time later if we redeside